soccer/ui/MainWindow.ui

```diff
@@ -1559,6 +1559,16 @@ p, li { white-space: pre-wrap; }
                   </widget>
                  </item>
                  <item>
+                  <widget class="QCheckBox" name="useDefenseCheckBox">
```

Comment:
It would be nice to have defense automatically turn off/on when you check/uncheck the box, w/o restarting the play, but thats more of a stretch goal.